src/components/action-toolbar/action-toolbar-multi-action-button/__spec__.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import ActionToolbar from './../action-toolbar';
+import ActionToolbarMultiActionButton from './action-toolbar-multi-action-button';
+import Button from './../../button';
+import { mount } from 'enzyme';
+
+describe('ActionToolbarMultiActionButton', () => {
+  let wrapper, button;
+  let selected = {};
+
+  beforeEach(() => {
+    wrapper = mount(
+      <ActionToolbar
+        actions={[]}
```

Comment:
This should be an object https://github.com/Sage/carbon/blob/c624df3e5e9d16d23915fbb3301134536f960407/src/components/action-toolbar/action-toolbar.js#L47

Reply:
I think the action-toolbar is wrong - it should be an array of objects right? Maybe we can fix that in this PR too?

Reply:
Sure, thought we had fixed that

Reply:
Sorry about the delayed reply. Do I need to change this?